site/pages/signer/authentication/social-login.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Oauth Authentication
+description: Authenticate a user using Oauth
+---
+
+# Social Login Authentication
+
+Email magic link authentication allows you to login and signup users using an Oauth provider, such as Google Sign-In or Facebook Login. You can also configure custom providers through Auth0.
+
+:::tip
+For setting up an account config, see the [Signer Quickstart](/signer/quickstart).
```

Comment:
this isn't super clear for a dev because (1) it's not clear this is a required step (2) they probably don't know what an account config is

```suggestion
First, follow the [Signer Quickstart](/signer/quickstart) to create a social policy in the dashboard.
```